docs/eventing/sources/apiserversource.md

```diff
@@ -0,0 +1,229 @@
+---
+title: "APIServerSource"
+linkTitle: "APIServerSource"
+weight: 31
+type: "docs"
+---
+
+![version](https://img.shields.io/badge/API_Version-v1beta1-red?style=flat-square)
+
+An APIServerSource brings Kubernetes API server events into Knative.
+
+## Installation
+
+The APIServerSource source type is enabled by default when you install Knative Eventing.
+
+## Example
+
+This example shows how to create an APIServerSource that listens to Kubernetes Events and
+send CloudEvents to a Knative Service.
+
+### Creating a namespace
+
+Create a new namespace called `apiserversource-example` by entering the following
+command:
+
+```shell
+kubectl create namespace apiserversource-example
+```
+
+### Creating the event display service
+
+In this step, you create one event consumer, `event-display` to verify that
+`APIServerSource` is properly working.
+
+To deploy the `event-display` consumer to your cluster, run the following
+command:
+
+```shell
+kubectl -n apiserversource-example apply -f - << EOF
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: event-display
+spec:
+  template:
+    spec:
+      containers:
+        - image: gcr.io/knative-releases/knative.dev/eventing-contrib/cmd/event_display
+EOF
```

Comment:
instead of creating a Knative service, let's use just a plain k8s service: https://knative.dev/development/eventing/sources/pingsource/

Reply:
Sounds good. This should be fixed in the new commit. Btw, there is an issue in the APIVersion of the sink in pingSource, it's fixed too.